qcodes/instrument_drivers/rigol/DG1062.py

```diff
@@ -182,8 +187,15 @@ def __init__(self, parent: 'DG1062', name: str, channel: int) ->None:
 
         self.add_parameter(
             "state",
-            set_cmd=f"OUTPUT{channel}:STATE {{}}",
             get_cmd=f"OUTPUT{channel}:STATE?",
+            set_cmd=f"OUTPUT{channel}:STATE {{}}",
+        )
+        
+        self.add_parameter(
+            "dcycle",
```

Comment:
could you rename this to `duty_cycle` for readability? good names give much more long-term benefit that saving on a couple of characters (it does depend on a particular case, but here - for sure).

Reply:
Done.